jest.conf/setup.visual.js

```diff
@@ -0,0 +1,72 @@
+import 'regenerator-runtime/runtime';
+import '@testing-library/jest-dom';
+import * as Aphrodite from 'aphrodite';
+import * as AphroditeNoImportant from 'aphrodite/no-important';
+
+// eslint-disable-next-line import/no-unresolved
+import 'mock-match-media/jest-setup';
```

Comment:
I think we could standarize better the naming/organization of the visual testing config files. We have a `setup.visual.js` and a `visual.index.js`, I think we could prefer one of both, if we want to prefix the `visual.` or put it before the `.js`. I would prefer to have both `visual.setup.js` and `visual.index.js` so they will be sorted together. Other option could be to have them in a subfolder, but this is something we can discuss.

Reply:
I guess we can go with having the prefix as 'visual' for such files. That'd indeed be better.